eZ/Publish/Core/Limitation/NewSectionLimitationType.php

```diff
@@ -166,4 +159,31 @@ public function valueSchema()
     {
         throw new \eZ\Publish\API\Repository\Exceptions\NotImplementedException(__METHOD__);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getCriterionByTarget(APILimitationValue $value, APIUserReference $currentUser, ?array $target): CriterionInterface
+    {
+        if ($this->doEvaluate($value, $target)) {
```

Comment:
Won't this always give matchAll() in cases people using PermissionCriterionResolver::getPermissionsCriterion forget to pass target(s)?

There won't be any warning or block of access in this case it will as far as I understand the change just give full access.